### DISCUSSION TITLE
change drag color

### DISCUSSION THREAD

**Discussion by briggssong** [sourcelair/xterm.js#4518] [Q&A]
i want change drag color.

maybe drag color chnaged opacity, right ? 
 
so if i changed theme 'background : #ffffff', it is hard to see.

plz check issue

**Comment by Tyriar** [comment]
Do you mean the selection? You control is via the selection* theme keys:

https://github.com/xtermjs/xterm.js/blob/a1e9105ce78154eafce3500ace119843a72c46f7/typings/xterm.d.ts#L291-L296
